builder.py

```diff
@@ -149,14 +149,10 @@ def BuildQueueCleanup(self, queue):
         queue.append('date')
         # Basic steps are:
         #   1. git stash (in each subproject)
-        #   2. git clean -fdx (in each subproject)
-        #   3. git fetch (in each subproject)
+        #   2. git fetch (in each subproject)
         queue.append('git stash')
         queue.append('git submodule foreach git stash')
```

Comment:
This one is **not** superfluous, as `git stash` only applies to the current repository.